Tests/LibWeb/Layout/input/flex/br-element-does-not-get-blockified-by-itself.html

```diff
@@ -1,4 +1,5 @@
-<!doctype><style>
+<!doctype html>
```

Comment:
this one has lowercase for DOCTYPE. that case difference doesn't seem relevant to the test case.